README.md

```diff
@@ -111,3 +119,20 @@ http://localhost:8888/?token=<UNIQUE-TOKEN>
 ```
 
 NOTE: This security token will change every time you start your Docker container.
+
+
+### Appendix: Build the Docker image
```

Comment:
I like this re-arrangement of the README to emphasize the fact that in general new users don't need to build the docker image. Thanks @BradNeuberg !

Reply:
absolutely! This is much clearer. Thanks!